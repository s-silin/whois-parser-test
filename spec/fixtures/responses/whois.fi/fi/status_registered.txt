domain.............: google.fi
status.............: Registered
created............: 30.6.2006 00:00:00
expires............: 4.7.2019 10:15:55
available..........: 4.8.2019 10:15:55
modified...........: 2.6.2018
holder transfer....: 20.10.2016
RegistryLock.......: no

Nameservers

nserver............: ns1.google.com [OK]
nserver............: ns2.google.com [Technical Error]
nserver............: ns3.google.com [Technical Error]
nserver............: ns4.google.com [Technical Error]
dnssec.............: unsigned delegation

Holder

name...............: Google Inc.
register number....: 3582691
address............: 1600 Amphitheatre Parkway
address............: 94043
address............: Mountain View
country............: United States of America
phone..............: +1.6502530000
holder email.......: 

Registrar

registrar..........: MarkMonitor Inc.
www................: www.markmonitor.com

Tech

name...............: Google Inc.
email..............: ccops@markmonitor.com

>>> Last update of WHOIS database: 2.8.2018 4:15:17 (EET) <<<


Copyright (c) Finnish Communications Regulatory Authority


