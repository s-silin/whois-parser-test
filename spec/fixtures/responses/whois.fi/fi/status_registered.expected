#domain
  %s == "google.fi"

#domain_id
  %s %ERROR{AttributeNotSupported}

#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2006-06-30}

#updated_on
  %s %CLASS{time}
  %s %TIME{2018-06-02}

#expires_on
  %s %CLASS{time}
  %s %TIME{2019-07-04 10:15:55}


#registrar
  %s %ERROR{AttributeNotSupported}

#registrant_contacts
  %s %ERROR{AttributeNotSupported}

#admin_contacts
  %s %ERROR{AttributeNotSupported}

#technical_contacts
  %s %ERROR{AttributeNotSupported}


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.google.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.google.com"
